### PR TITLE
Implement order confirmation page component

### PR DESCRIPTION
**Pull request by AlexeyKaryka**
## This PR is a:

[ X ] New feature
[ ] Enhancement/Optimization
[ ] Refactor
[ ] Bugfix
[ ] Test for existing code
[ ] Documentation

## Summary

Will be changed order confirmation component(named Exit), button for creating account is not yet active.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento-research/pwa-studio?pullRequest=358) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento-research/pwa-studio?pullRequest=358) before we can accept your contribution.<br/>

File: packages/venia-concept/src/components/CreateAccountButton/createAccountButton.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { shape, string } from 'prop-types';
+import PropTypes from 'prop-types';
+import classify from 'src/classify';
+import Button from 'src/components/Button';
+
+import defaultClasses from './createAccountButton.css';
+
+class CreateAccountButton extends Component {
```
Comment (Starotitorov) [inline packages/venia-concept/src/components/CreateAccountButton/createAccountButton.js]: Please, right now, create a Button component inside order confirmation page folder, which will receive children property. Add this button to order confirmation page component.

```
class CustomButton extends Component {
    render() {
        const { onClick, children } = this.props;
        return (
            <Button classes={{ root: classes.root }} onClick={onClick}>
                { children }
            </Button>
        );
    }
}
```

File: packages/venia-concept/src/components/CreateAccountButton/createAccountButton.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.createAccountButtonRoot {
```
Comment (Starotitorov) [inline packages/venia-concept/src/components/CreateAccountButton/createAccountButton.css]: You just copied and inserted css from resetButton.css

File: packages/venia-concept/src/components/Checkout/receipt.js [hunk 3]
```diff
@@ -12,17 +13,33 @@ class Exit extends Component {
...
     render() {
-        const { classes, resetCheckout } = this.props;
+        const { classes, resetCheckout, orderId } = this.props;
 
         return (
             <div className={classes.root}>
-                <div className={classes.body}>Thank you for your order!</div>
-                <div className={classes.footer}>
+                <div className={classes.body}>
+                    <h2 className={classes.headerText}>
```
Comment (Starotitorov) [inline packages/venia-concept/src/components/Checkout/receipt.js]: We do not have css for this className. In general, we need to discuss our approach with classify HOC.

File: packages/venia-concept/src/components/CreateAccountButton/createAccountButton.js (hunk 1 above)
Comment (Starotitorov) [reply packages/venia-concept/src/components/CreateAccountButton/createAccountButton.js]: In future we need to think about adding something like theme property to common Button component. In this case we will be able to write something like this

```
<Button theme="dark">...</Button>
```

File: packages/venia-concept/src/components/Checkout/receipt.js [hunk 4]
```diff
@@ -12,17 +13,33 @@ class Exit extends Component {
...
+                        Thank you for your purchase!
+                    </h2>
+                    <div className={classes.textBlock}>
+                        Your order # is <span className={classes.orderId}>{orderId}</span><br/>
+                        We\'ll email you an order confirmation with details and tracking info
+                    </div>
                     <ResetButton resetCheckout={resetCheckout} />
```
Comment (Starotitorov) [inline packages/venia-concept/src/components/Checkout/receipt.js]: We do not need reset button here. Just implement a CustomButton component for our page, see comment below.

**Pull request merged**